src/components/template-gallery/TemplateGalleryItem.tsx

```
@@ -0,0 +1,66 @@
+          <h3 className="text-sm font-medium text-white truncate">
+            {template.title}
+          </h3>
+          {template.description && (
+            <p className="text-xs text-gray-300 mt-1 line-clamp-2">
+              {template.description}
+            </p>
+          )}
+        </div>
+
+        {/* Action Button */}
+        <button className="flex cursor-pointer items-center gap-2 text-sm text-white hover:underline self-end">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)